tasks/keepalived_install.yml

```diff
@@ -81,3 +94,9 @@
   tags:
     - keepalived-config
     - keepalived-prevent-start
+
+- name: "Reload systemd if we deleted service mask"
```

Comment:
This should probably be a handler. When the package is changed, we should do a systemd daemon_reload.

On top of that it won't work with ansible < 2.4. It only passed my automatic testing because I test with ansible 2.3 and ubuntu 14.